.github/scripts/userdata.sh

```diff
@@ -152,9 +160,11 @@ sudo sed -i 's/https/http/' /etc/kibana/kibana.yml
 EOF
 else
 sed -i "s/^echo \"cluster.name.*/echo \"cluster.name \: odfe-$ODFE_VER-$1-noauth\" \>\> config\/elasticsearch.yml/g" $REPO_ROOT/userdata_$1.sh
-sed -i "/echo \"network.host/a echo \"opendistro_security.disabled: true\" \>\> config\/elasticsearch.yml" $REPO_ROOT/userdata_$1.sh
 cat <<- EOF >> userdata_$1.sh
 sudo rm -rf plugins/opendistro_security
+sed -i /^opendistro_security/d config/elasticsearch.yml
+cd /opendistroforelasticsearch-kibana/
+sudo rm -rf plugins/opendistro_security
```

Comment:
Duplicate of line 164?

Reply:
Good catch.

Reply:
One is removing from elasticsearch folder and other from kibana plugins' folder

Reply:
Ah, I see. Thanks!